packages/components/src/sandbox/style.scss

```
@@ -1,3 +1,6 @@
 .components-sandbox {
 	overflow: hidden;
 }
+iframe.components-sandbox {
```

Nitpick: Don't we usually add an extra line between rule blocks?
```suggestion

iframe.components-sandbox {
```